commit 79de8aa31cf60929218417da6b05a108fd9412d7
Author: Jonathan Riddell <jr@jriddell.org>
Date:   Fri Jan 10 13:33:41 2014 +0000

    rename kjs developer binary to kjs5

diff --git a/src/kjs/CMakeLists.txt b/src/kjs/CMakeLists.txt
index 63be284..8b88cca 100644
--- a/src/kjs/CMakeLists.txt
+++ b/src/kjs/CMakeLists.txt
@@ -210,7 +210,7 @@ install(TARGETS KF5JS EXPORT KF5JSTargets ${INSTALL_TARGETS_DEFAULT_ARGS})
 set(kjs_SRCS kjs.cpp)
 
 #required by win32 see  kdelibs/cmake/modules/kde4Macros.cmake kde4_add_manifest
-set (kjs_bin_OUTPUT_NAME kjs)
+set (kjs_bin_OUTPUT_NAME kjs5)
 
 # 'kjs_bin' because cmake doesn't like having a lib and app with the same name
 # This is a developer tool, not intended for normal user installs
